```
builder for '/nix/store/ydy2awlgk4w4lg15y6fcwsmh2lkyh6cc-python3.8-slackclient-1.2.1.drv' failed with exit code 1; last 10 log lines:                                                         Check whether the following modules can be imported: slack                                                                                                                                  Traceback (most recent call last):                                                                                                                                                            File "<string>", line 1, in <module>                                                                                                                                                        File "<string>", line 1, in <lambda>                                                                                                                                                        File "/nix/store/wkw6fsjasr7jbbrlakxxpbiapa8hws42-python3-3.8.7/lib/python3.8/importlib/__init__.py", line 127, in import_module                                                              return _bootstrap._gcd_import(name[level:], package, level)                                                                                                                               File "<frozen importlib._bootstrap>", line 1014, in _gcd_import                                                                                                                             File "<frozen importlib._bootstrap>", line 991, in _find_and_load                                                                                                                           File "<frozen importlib._bootstrap>", line 973, in _find_and_load_unlocked                                                                                                                ModuleNotFoundError: No module named 'slack'                                                                                                                                              cannot build derivation '/nix/store/niqcq7bb5479c1djc5a2fvw76skcbvmy-errbot-6.1.1.drv': 1 dependencies couldn't be built                                                                    cannot build derivation '/nix/store/5v09380mrgxaymmnrmgplaasg1rwmq16-review-shell.drv': 1 dependencies couldn't be built                                                                    error: --- Error ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- nixbuild of '/nix/store/5v09380mrgxaymmnrmgplaasg1rwmq16-review-shell.drv' fail
```

--- a/pkgs/development/python-modules/datashader/default.nix
+++ b/pkgs/development/python-modules/datashader/default.nix
@@ -76,11 +76,6 @@ buildPythonPackage rec {
     nbconvert
   ];
 
-  postConfigure = ''
-    substituteInPlace setup.py \
-      --replace "'numba >=0.37.0,<0.49'" "'numba'"
-  '';
-
   # dask doesn't do well with large core counts
   checkPhase = ''
     pytest -n $NIX_BUILD_CORES datashader -k 'not dask.array'

--- a/pkgs/development/python-modules/j2cli/default.nix
+++ b/pkgs/development/python-modules/j2cli/default.nix
@@ -26,7 +26,7 @@ buildPythonPackage rec {
       J2Cli is a command-line tool for templating in shell-scripts,
       leveraging the Jinja2 library.
     '';
-    maintainers = with maintainers; [ rushmorem ];
+    maintainers = with maintainers; [ rushmorem SuperSandro2000 ];
   };
 
 }

--- a/pkgs/tools/misc/duf/default.nix
+++ b/pkgs/tools/misc/duf/default.nix
@@ -2,26 +2,24 @@
 
 buildGoModule rec {
   pname = "duf";
-  version = "0.5.0";
+  version = "0.6.0";
 
   src = fetchFromGitHub {
     owner = "muesli";
     repo = "duf";
     rev = "v${version}";
-    sha256 = "0n0nvrqrlr75dmf2j6ja615ighzs35cfixn7z9cwdz3vhj1xhc5f";
+    sha256 = "sha256-Wm3gfir6blQFLLi+2bT5Y/5tf7qUxEddJQ7tCYfBGgM=";
   };
 
-  dontStrip = true;
+  vendorSha256 = "0icxy6wbqjqawr6i5skwp1z37fq303p8f95crd8lwn6pjjiqzk4i";
 
-  vendorSha256 = "1jqilfsirj7bkhzywimzf98w2b4s777phb06nsw6lr3bi6nnwzr1";
-
-  buildFlagsArray = [ "-ldflags=" "-X=main.Version=${version}" ];
+  buildFlagsArray = [ "-ldflags=" "-s -w -X=main.Version=${version}" ];
 
   meta = with lib; {
     homepage = "https://github.com/muesli/duf/";
     description = "Disk Usage/Free Utility";
     license = licenses.mit;
     platforms = platforms.unix;
-    maintainers = with maintainers; [ petabyteboy penguwin ];
+    maintainers = with maintainers; [ petabyteboy penguwin SuperSandro2000 ];
   };
 }

